lib/plugins/install/install.js

```diff
@@ -30,6 +37,30 @@ class Install {
       'install:install': () => BbPromise.bind(this)
         .then(this.install),
     };
+
+    this.renameService = (name, servicePath) => new BbPromise((resolve) => {
+      const serviceFile = path.join(servicePath, 'serverless.yml');
+      const packageFile = path.join(servicePath, 'package.json');
+      BbPromise.all([
+        readFile(serviceFile, 'utf-8')
+          .then(contents =>
+            contents.replace(/service\s*:.+/gi, (match) => {
+              const fractions = match.split('#');
+              fractions[0] = `service: ${name}`;
+              return fractions.join(' #');
+            }))
+          .then(contents => writeFile(serviceFile, contents)),
+        this.serverless.utils.readFile(packageFile)
+          .then(contents => Object.assign(contents, { name }))
+          .then(contents => this.serverless.utils.writeFile(packageFile, contents)),
+      ])
+        .then(() => resolve(` as "${name}"`))
+        .catch((error) => {
```

Comment:
Or you could let Serverless catch the error and remove the `catch` block altogether.